distro/adaptation/centos

```diff
@@ -331,7 +331,7 @@ libsm6: libSM
 libsmbclient-dev: libsmbclient-devel
 libsnappy-dev: snappy-devel
 libsnappy1v5: snappy
-libsndio7.0: sndio
+libsndio7.0:
```

Comment:
thanks, is it possible to do this in adaptation/centos-9 only, as I'm not sure whether sndio exists on old centos